en/Releases/enebular-edge-agent/1.0.1.md

```diff
@@ -0,0 +1,24 @@
+---
+lastUpdated: 2018-05-25
+---
+
+# 1.0.1
+
+In 1.0.1 release, Updated Mbed Cloud Client to 1.3.1.1 General Availability (GA).
+
+### New
+
+N/A
+
+### Fixed
+
+* Fixed network disconnection issue when running for a long time due to exclusive control of communication processing of Wi-Fi module by updating Wi-Fi module's library
```

Comment:
(This might be more easier to understand)

Network disconnected when running an agent for extended periods of time. 
This was due to the Wi-Fi's module taking full control of the communication processing. 
Issue solved by updating Wi-Fi module.

Reply:
ありがとうございます！
updateしたのはWi-Fi moduleのライブラリなのでそこだけ直しました